sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -98,6 +93,10 @@ def close(self):
         self.__exit__()
 
     def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
+        from .pipeline import Pipeline
```

Comment:
Let's keep this one at the root, we need a Pipeline in all circumstances

Reply:
If users want to use their customized pipeline by passing pipeline kwargs, we don't need to import pipeline, right?

Reply:
We call build_pipeline only when user does not provide a pipeline, so we build a "default" one.

Reply:
fair enough :)